src/base/logger.cpp

```diff
@@ -72,7 +76,7 @@ void Logger::freeInstance()
 void Logger::addMessage(const QString &message, const Log::MsgType &type)
 {
     QWriteLocker locker(&m_lock);
-    const Log::Msg msg = {m_msgCounter++, QDateTime::currentMSecsSinceEpoch(), type, message.toHtmlEscaped()};
```

Comment:
I think this was done because of the use of HTML based formatting in `LogWidget` which is not required anymore.

Reply:
Do you mean now it is all plain text and no more coloring for different `MsgType`?

Reply:
coloring is handled by model, so we don't do HTML formatting any more, previously we're using QLabel with html formatting.